diff --git a/config.js b/config.js
index 0a6e330976c44fbe06a94a538e6e3c5661c8e2d6..c831ce489f729227eae58474227301d5370d851f 100644
--- a/config.js
+++ b/config.js
@@ -3,7 +3,9 @@ const { version } = require('./package');
 
 module.exports = {
   /* eslint-disable prefer-template -- for better formatting */
-  banner: '/**\n' +
+  banner: '// @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt MIT\n' +
+          '// @ts-nocheck\n' +
+          '/*\n' +
           ' * core-js ' + version + '\n' +
           ' * © 2014-2024 Denis Pushkarev (zloirock.ru)\n' +
           ' * license: https://github.com/zloirock/core-js/blob/v' + version + '/LICENSE\n' +
diff --git a/index.js b/index.js
index e5f8dcb053e2375aa2795e8dacaa5970babdfc92..5fabdfb98c3812613fac89bf10001e6261317335 100644
--- a/index.js
+++ b/index.js
@@ -51,8 +51,6 @@ module.exports = async function ({
         mode: 'none',
         node: {
           global: false,
-          process: false,
-          setImmediate: false,
         },
         entry: list.map(it => require.resolve(`core-js/modules/${ it }`)),
         output: {
